commands/system/help.js

```diff
@@ -1,83 +1,162 @@
 const Command = require(`${process.cwd()}/base/Command.js`);
 const { MessageEmbed } = require("discord.js");
+const EMOJIS = ["⏮", "◀", "⏹", "▶", "⏭", "🔢"];
 const perpage = 10;
-
 class Help extends Command {
   constructor(client) {
     super(client, {
       name: "help",
       description: "Get help on a command, command category, or a setting",
       extended: "This command will display all available commands for your permission level, with the additonal option of getting per command information when you run 'help <command name>'.",
       category: "System",
-      usage: "help <category/command/setting> [page-num]",
-      aliases: ["h", "halp", "commands"]
+      usage: "help <category/command/setting>",
+      aliases: ["h", "halp", "commands"],
     });
-  }
 
-  async run(message, [type, page], level) {
+    this.pages = async (message, helpmessage, pagenumber, sorted, type, level, reactions, direction) => {
+      let n = 0; // eslint-disable-line no-unused-vars
```

Comment:
If you're not using this, don't use this.

Reply:
For some reason, I noticed this too while I was toying around with his version. It *is* required, but if I were to remove it, parts want to break..